apiary_v2.apib

```diff
@@ -1231,4 +1231,670 @@ the listed orders by their creation date.
+
+    + Body
+
+          {
+              "code": "not_found",
+              "message": "Resource not found",
+          }
+
++ Response 405 (application/json)
+
+    + Body
+
+          {
+              "code": "method_not_allowed",
+              "message": "Method used is not allowed",
+          }
+
++ Response 406 (application/json)
+
+    + Body
+
+          {
+              "code": "not_acceptable",
+              "message": "Could not satisfy the request Accept header.",
+          }
+
++ Response 415 (application/json)
+
+    + Body
+
+          {
+              "code": "unsupported_media_type",
+              "message": "Unsupported media type in request.",
+          }
+
+## List Jobs [/factory.unmade.com/v2/orders/{order_id}/jobs{?page}]
+## List Jobs[GET]
+
+This endpoint can be used to retrieve a paginated list of the individual jobs associated with a given production order.
+
++ Parameters
+    + order_id (string) - ID of the Order in the form of a uuid
+    + page (optional, number) - Page number
+
++ Request List Order Items
+
+    + Headers
+
+            Authorization: Token ABCDEF
+
++ Response 200 (application/json)
+
+    + Body
+
+          {}
+
+## Retrieve Job [/factory.unmade.com/v2/orders/{order_id}/job/{job_id}]
+## Retrieve Job[GET]
+
+This endpoint can be used to retrieve data for a specific job in a given order.
+
++ Parameters
+    + order_id (string) - ID of the Order in the form of a uuid
+    + job_id (string) - ID of the job in the form of a uuid
+
++ Request Retrieve Order Item
+
+    + Headers
+
+            Authorization: Token ABCDEF
+
++ Response 200 (application/json)
+
+    + Body
+
+          {}
+
+## Set Job In Production [/factory.unmade.com/v2/orders/{order_id}/job/{job_id}/set_in_production]
+## Set In Production[POST]
+
+If a given order consists of a large number of jobs, production for these jobs may begin at different times. In such a 
+case, it is possible to inform Unmade that individual jobs have begun production using this endpoint. This updates the 
+state of this job's order to `in_production`, but does not affect the state of the other jobs in this order.
+
++ Parameters
+    + order_id (string) - ID of the Order in the form of a uuid
+    + job_id (string) - ID of the job in the form of a uuid
+
++ Request Retrieve Order Item
+
+    + Headers
+
+            Authorization: Token ABCDEF
+
++ Response 200 (application/json)
+
+    + Body
+
+          {}
+
+## Set Job Ready to Ship [/factory.unmade.com/v2/orders/{order_id}/job/{job_id}/set_ready_to_ship]
+## Set Ready to Ship[POST]
+
+If a given order consists of a large number of jobs, production for these jobs may be completed at different times. In 
+such a case, it is possible to inform Unmade that individual jobs have completed production and are ready to ship using 
+this endpoint. This updates the state of this job's order to `ready_to_ship`, but does not affect the state of the other
+jobs in this order.
```

Comment:
This means that if all are ready to ship the order won't be ready to ship unless they told us?

Reply:
Oh, you're right. This is wrong. It will update the order to ready to ship after **all** the jobs have been marked